tokio/src/net/windows/named_pipe.rs

```diff
@@ -1687,8 +1683,14 @@ impl ServerOptions {
     /// ```
     pub fn new() -> ServerOptions {
         ServerOptions {
-            open_mode: windows_sys::PIPE_ACCESS_DUPLEX | windows_sys::FILE_FLAG_OVERLAPPED,
-            pipe_mode: windows_sys::PIPE_TYPE_BYTE | windows_sys::PIPE_REJECT_REMOTE_CLIENTS,
```

Comment:
The `PIPE_ACCESS_DUPLEX` option seems to have disappeared?

Reply:
[`PIPE_ACCESS_DUPLEX`](https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createnamedpipea) is `PIPE_ACCESS_INBOUND | PIPE_ACCESS_OUTBOUND`.